composer.json

```diff
@@ -12,37 +12,36 @@
         }
     ],
     "require": {
-        "php": "^5.5.0",
-        "symfony/framework-bundle": ">=2.1,<3.0-dev",
-        "doctrine/common": ">=2.3,<3.0",
-        "jms/di-extra-bundle": ">=1.1,<2.0"
+        "php": "^5.5|^7.0",
+        "symfony/framework-bundle": "^2.3|^3.0",
```

Comment:
Symfony 2.1 and 2.2 are not supported anymore so IMO we should not continue support them too.